frontend/src/pages/add-blog.tsx

```diff
@@ -122,125 +122,145 @@ function AddBlog() {
+            </div>
+            <textarea
+              name="description"
+              placeholder="Start writing here&hellip;"
+              rows={5}
+              className="w-full rounded-lg bg-slate-200 p-3 placeholder:text-sm placeholder:text-light-tertiary dark:bg-dark-card dark:text-slate-50 dark:placeholder:text-dark-tertiary"
+              value={formData.description}
+              onChange={handleInputChange}
+            />
           </div>
-        </div>
 
-        <div className="mb-4 flex items-center">
-          <label className="flex items-center">
-            <span className="text-base font-medium text-gray-800 dark:text-white">
-              Featured Post
-            </span>
+          <div className="mb-2">
+            <div className="px-2 py-1 font-medium text-light-secondary dark:text-dark-secondary">
+              Author name <Asterisk />
+            </div>
             <input
-              type="checkbox"
-              name="isFeaturedPost"
-              className="ml-2 h-5 w-5 rounded border text-indigo-600 focus:ring-indigo-400"
-              checked={formData.isFeaturedPost}
-              onChange={handleCheckboxChange}
+              type="text"
+              name="authorName"
+              placeholder="Shree Sharma"
+              className="w-full rounded-lg bg-slate-200 p-3 placeholder:text-sm placeholder:text-light-tertiary dark:bg-dark-card dark:text-slate-50 dark:placeholder:text-dark-tertiary"
+              value={formData.authorName}
+              onChange={handleInputChange}
             />
-          </label>
-        </div>
+          </div>
 
-        <button
-          type="submit"
-          className="flex w-full items-center justify-center rounded-lg bg-black p-2 text-base text-white hover:bg-gray-800 dark:bg-white dark:text-black md:w-fit"
-        >
-          Create Blog
-        </button>
-      </form>
-      <ModalComponent
-        selectedImage={selectedImage}
-        handleImageSelect={handleImageSelect}
-        handleSelector={handleselector}
-        setModal={setmodal}
-        modal={modal}
-      />
+          <div className="px-2 py-1 font-medium text-light-secondary dark:text-dark-secondary">
+            Blog cover image
+            <span className="text-xs tracking-wide text-dark-tertiary">
+              &nbsp;(jpg/png/webp)&nbsp;
+            </span>
+            <Asterisk />
+          </div>
+          <div className="mb-4 flex justify-between gap-4">
+            <div className="w-10/12">
+              <input
+                type="url"
+                id="imagelink"
+                name="imageLink"
+                placeholder="https://&hellip;"
+                autoComplete="off"
+                className="w-full rounded-lg bg-slate-200 p-3 placeholder:text-sm placeholder:text-light-tertiary dark:bg-dark-card dark:text-slate-50 dark:placeholder:text-dark-tertiary"
+                value={formData.imageLink}
```

Comment:
Did that here: https://github.com/krishnaacharyaa/wanderlust/pull/67/commits/9f5a0d36b69982c57585fb2757c00da3ffcebcde. Didn't work.
I think it doesn't happen for `className` attributes. It's a hit or miss at least.
Because iirc, I had to break the line manually in a personal repo of mine.

Reply:
Yeah, i can even see it in `frontend/tsconfig.json`, can we do something for that? 

p.s : https://stackoverflow.com/questions/71377814/prettier-code-formatting-is-not-splitting-classnames-in-jsx-or-html got this from the net. We can use something which suits us the best

Reply:
The Headwind plugin that's mentioned in the answer is not maintained since 2021, and it will definitely conflict with the official tailwind plugin that we're using.
I'll look into it.

Reply:
`word-wap` will be the only option then, yeah sure, it's not of top priority